client/app/students/new-student/new-student.html

```diff
@@ -33,9 +34,36 @@
                     <p ng-message="pattern">Class must be formatted כיתה-מספר</p>
                 </div>
             </md-input-container>
+            <md-input-container>
+                <label>Social Grade</label>
+                <input name="social" ng-model="student.social"  ng-pattern='/^([1-4])$/' required/>
```

Comment:
Maybe this should be `type="number"`? it will automatically convert it to a number

Reply:
changed at new-student.html and in the edit option at student.html.